### PR TITLE
Add support for `--graph=file.html`

### PR DESCRIPTION
**Pull request by styfle**
This PR adds support for HTML graphs using `viz.js`.

This is nice because you don't need to install [graphviz](https://graphviz.org/) to see the graph and you get zoom for free.

### CODE REVIEW

File: cli/internal/run/run.go [hunk 1]
```diff
@@ -612,9 +612,32 @@ func (c *RunCommand) Run(args []string) int {
...
+      <html>
+      <head>
+        <meta charset="utf-8">
+        <title>Graph</title>
+      </head>
+      <body>
+        <script src="https://cdn.jsdelivr.net/npm/viz.js@2.1.2-pre.1/viz.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/viz.js@2.1.2-pre.1/full.render.js"></script>
+        <script>`)
+      f.WriteString("const s = " + graphString + ";new Viz().renderSVGElement(s).then(el => document.body.appendChild(el)).catch(e => console.error(e));")
```
Comment (ernestd) [inline cli/internal/run/run.go:631]: do we run linters on go code?

Comment (jaredpalmer) [reply cli/internal/run/run.go]: fixed

**Review by jaredpalmer — APPROVED** [review]

File: cli/internal/run/run.go [hunk 2]
```diff
@@ -613,9 +612,40 @@ func (c *RunCommand) Run(args []string) int {
 			Verbose:    true,
 			DrawCycles: true,
 		}))
+		ext := filepath.Ext(runOptions.dotGraph)
+		log.Println(ext)
```
Comment (styfle) [inline cli/internal/run/run.go:616]:
```suggestion
```

@jaredpalmer Did you intend to leave this? Should probably be removed

Comment (styfle) [reply cli/internal/run/run.go]: I removed it in 42583bd

**Review by jaredpalmer — APPROVED** [review]

**Pull request merged**